client/src/app/site/services/auth-check.service.ts

```diff
@@ -0,0 +1,75 @@
+import { Injectable } from '@angular/core';
+import { Data } from '@angular/router';
+import { OML } from 'src/app/domain/definitions/organization-permission';
+import { Permission } from 'src/app/domain/definitions/permission';
+import { Settings } from 'src/app/domain/models/meetings/meeting';
+
+import { ActiveMeetingService } from '../pages/meetings/services/active-meeting.service';
+import { MeetingSettingsService } from '../pages/meetings/services/meeting-settings.service';
+import { OperatorService } from './operator.service';
+import { RerouteService } from './reroute.service';
+
+export enum AuthorizationStatus {
+    Authorized,
+    NoPerms,
+    NotLoggedIn
```

Comment:
Please use an own enumeration `AuthenticationStatus` (this could be in another service, like the `AuthService`).
I like opposites more: `Authenticated` and `Unauthenticated`.